src/cephadm/cephadm

```diff
@@ -3077,8 +3077,11 @@ class CephContainer:
 @infer_image
 def command_version(ctx):
     # type: (CephadmContext) -> int
-    out = CephContainer(ctx, ctx.image, 'ceph', ['--version']).run()
-    print(out.strip())
+    try:
+        out = CephContainer(ctx, ctx.image, 'ceph', ['--version']).run()
+        print(out.strip())
+    except Exception:
+        pass
```

Comment:
though, we should not `return 0` if that fails.

Reply:
podman return code is passed along 
![image](https://user-images.githubusercontent.com/22037319/108548831-5924ce80-72ba-11eb-98be-97ac1cce8328.png)